plugin/package.json

```diff
@@ -55,17 +55,20 @@
     "pathe": "^0.2.0",
     "pretty-bytes": "^5.6.0",
     "semver": "^7.3.5",
-    "tempy": "^1.0.0"
+    "tempy": "^1.0.0",
+    "uuid": "^8.3.2"
   },
   "devDependencies": {
     "@gatsbyjs/reach-router": "^1.3.6",
     "@netlify/build": "^27.1.1",
     "@types/fs-extra": "^9.0.12",
     "@types/multer": "^1.4.7",
     "@types/semver": "^7.3.9",
+    "chance": "^1.1.8",
```

Comment:
Added in order to introduce randomized values in test cases. Package documentation can be found [here](https://chancejs.com/)